src/components/include/cos_kernel_api.h

```diff
@@ -137,9 +137,9 @@ int cos_thd_mod(struct cos_compinfo *ci, thdcap_t c, void *tls_addr); /* set tls
 
 int cos_asnd(asndcap_t snd, int yield);
 /* returns non-zero if there are still pending events (i.e. there have been pending snds) */
-int cos_rcv(arcvcap_t rcv, rcv_flags_t flags, int *rcvd);
+int cos_rcv(arcvcap_t rcv, rcv_flags_t flags, tcap_time_t timeout, int *rcvd);
```

Comment:
I thought it's an useful utility for some threads that they either wait for an event or for a timeout.